<!--
ROB599 Homwork 3 - timer node sends messages to itself.
Jake Anderson
Oct 24, 2020
-->


<launch>

	<!-- Start the timer node. -->
	<remap from="/noiser/laser_scan_in" to="base_scan"/>
	<node pkg="rob599_hw3" name="laser_noise" type="noiser.py" output="screen" >
	</node>


</launch>
